boards/xiao_m0/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.11.0
+
+- update gpio and sercom dependencies to v2
+- create shared_i2c example with I2C bus used for SSD1306 OLED screen and accelerometer
+- create ssd1306_i2c example with basic usage of OLED screen for animation with I2C
+- update blink example to be more readable for newcomers
+- clean up usb_echo example (and extend of blinking on data transfer)
+
 # Unreleased
 
 - remove extraneous `embedded-hal` dependencies from BSPs
```

Comment:
If we're doing a release, move all items in `# Unreleased` to your release and put an empty `# Unreleased` at the top.